### DISCUSSION TITLE
Security Onion Setup Issue (2.3.10)

### DISCUSSION THREAD

**Discussion by grizzlycode** [Security-Onion-Solutions/securityonion#2082] [Unsupported Versions] [answered]
I see from Reddit (r/securityonion) that they are directing support requests over here. I have no problem with that however, I think it makes that sub-Reddit a moot point to even exist as that is typically 80% of what people use Reddit for when it concerns a product or IT subject. Anyways, here is my issue: 

So I pulled down the latest ISO (2.3.10) and tried installing the "IMPORT" node for a standalone workstation however, after reboot there is no desktop as it shows in link below. 

I can get to the web interface which is what I would expect if I set it up in another mode however, I was expecting a desktop to conduct analysis. 

Anyone else having this issue? 

Setup link:
https://blog.securityonion.net/2020/10/security-onion-2-has-reached-general.html

**Comment by jertel** [comment, marked as answer]
The import install does not come with a desktop. 

Prior to the first of those desktop screenshots there's a caption that mentions the ability to install the optional analyst desktop, via the `so-analyst-install` command.

![image](https://user-images.githubusercontent.com/16639611/100152190-4b54de00-2e70-11eb-8e1d-6b80cb1fcf1e.png)

**Reply by grizzlycode** [reply]
Thanks that was an oversight on my part. However, I ran the command (`so-analyst-install`) and rebooted and I still have no desktop. I captured the CLI install and I have a ton of errors. 

[analyst_install.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5592998/analyst_install.txt)

**Reply by jertel** [reply]
The current version of the so-analyst-install script requires Internet access, so if you're in a disconnected network / airgap environment, which seems to be the case due to the inability to access those URLs, it would explain the errors. If that's not the case then you may need to investigate why that VM is unable to reach out to the Internet, or those specific centos.org URLs.

**Reply by grizzlycode** [reply]
It was a DNS error during setup in a VM. Thanks for the quick responses!
